src/schemix_orm/main.py

```
@@ -0,0 +1,188 @@
+
+    # Option 2: Manual setup (if you need more control)
+    # sqlite_conn = await aiosqlite.connect(":memory:")
+    # await sqlite_conn.execute("PRAGMA foreign_keys = ON")
+    # await sqlite_conn.commit()
+    # connection = SQLiteConnection(sqlite_conn)
+
+    db = Database(connection, [Users, Posts])
+
+    # For table creation, we still need the raw connection
+    # This will be improved when we add schema generation
+    sqlite_conn = connection._conn
```

Pylint (3.3.7)

[warning] Access to a protected member _conn of a client class

(W0212)